src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -259,10 +259,9 @@ class PublicationReleasePage extends Component<Props> {
           <AccordionSection heading="National statistics" headingTag="h3">
             <p className="govuk-body">
               The United Kingdom Statistics Authority designated these
-              statistics as National Statistics in <a href="#">Month Year</a> in
-              accordance with the Statistics and Registration Service Act 2007
-              and signifying compliance with the Code of Practice for
-              Statistics.
+              statistics as National Statistics in 2011 in accordance with the
```

Comment:
The "Year" will need to come from an api property

Reply:
There is now a placeholder "[INSERT MONTH YEAR]" in the content to be addressed later. So I'll mark this as resolved.